src/ecc_key.c

```diff
@@ -1280,6 +1284,64 @@ int ec_key_sec2prot(struct zpc_ec_key *ec_key, enum ec_key_sec sec)
 	return 0;
 }
 
+int ec_key_clr2prot(struct zpc_ec_key *ec_key, const unsigned char *privkey,
+					unsigned int privlen)
+{
+	struct pkey_kblob2pkey3 io;
+	unsigned char buf[sizeof(struct clearkeytoken) + 80];
+	struct clearkeytoken *clrtok = (struct clearkeytoken *)&buf;
+	int rc;
+
+	memset(buf, 0, sizeof(buf));
+	clrtok->version = 0x02;
```

Comment:
I guess there is no define for that 0x02 ?

Reply:
No, actually not. It's just the required version as defined in the header pkey.h